sim/core/spell_mod.go

```diff
@@ -422,47 +422,47 @@ var spellModMap = map[SpellModType]*SpellModFunctions{
 }
 
 func applyDamageDonePercent(mod *SpellMod, spell *Spell) {
-	spell.DamageMultiplier *= 1 + mod.floatValue
+	spell.DamageMultiplier *= mod.floatValue
 }
 
 func removeDamageDonePercent(mod *SpellMod, spell *Spell) {
-	spell.DamageMultiplier /= 1 + mod.floatValue
+	spell.DamageMultiplier /= mod.floatValue
 }
 
 func applyDamageDoneAdd(mod *SpellMod, spell *Spell) {
-	spell.DamageMultiplierAdditive += mod.floatValue
+	spell.DamageMultiplierAdditivePct += +mod.intValue
```

Comment:
Is this intended as it's only done here? `+= +mod.intValue`

Reply:
Oh no that's a typo